### PR TITLE
Rails 5 support

### PR DESCRIPTION
**Pull request by benlovell**
Best reviewed commit-by-commit.

This was pretty straight-forward, but does rely on some unreleased code. This
shouldn't be a blocker for merge. I'll keep track of the unreleased
dependencies we've pinned.

### CODE REVIEW

**Comment by benlovell** [comment]
There are a few outstanding deprecations being spewed by `jasmine-rails`. I have a searls/jasmine-rails#198 and working on a fix for the outstanding `alias_method_chain` ones.

File: Gemfile [hunk 1]
```diff
@@ -16,7 +17,7 @@ gem 'plek'
 gem 'puma'
 gem 'foreman'
 gem 'sidekiq'
-gem 'sinatra', require: false
+gem 'sinatra', '2.0.0.beta2', require: false
```
Comment (benbarnett) [inline Gemfile]: This might be a 😳 moment but are we actually using sinatra?

Comment (Guntrisoft) [inline Gemfile]: https://github.com/mperham/sidekiq/wiki/Monitoring

Comment (benbarnett) [inline Gemfile]: Ah, the sense is made.

**Comment by Guntrisoft** [comment]
Looks good from my 'very' limited knowledge ✅

**Comment by benbarnett** [comment]
Same here 👍

**Pull request merged**